integration/testcmd/dhclient/main.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+
+	"github.com/u-root/webboot/pkg/dhclient"
+)
+
+var (
+	timeout = flag.Int("timeout", 15, "Lease timeout in seconds")
+	retry   = flag.Int("retry", 5, "Max number of attempts for DHCP clients to send requests. -1 means infinity")
+	ipv4    = flag.Bool("ipv4", true, "use IPV4")
+	ipv6    = flag.Bool("ipv6", true, "use IPV6")
```

Comment:
You should either use these four flags or remove them.

Reply:
I originally did not have them since I did not use any. However, Gan suggested that it would not hurt to add flags to the file main.go so you can add tests for different configurations to test in the future.

Reply:
I just had a discussion with you about this because I was slightly confused what you meant :) However, I get what you mean now! Thank you